core/ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -402,7 +434,17 @@
       ],
       "name": "default",
       "defaultEnv": "default",
-      "description": null
+      "description": null,
```

Comment:
@benoitf yes, it is not easy to find a command that works for all users. But at least this is smth to start with and the command can be edited if needed. I think it's better to remove it at all for node as there can be plenty of options here.